🔎 checking possible migrations..

attention => Storybook now collects completely anonymous telemetry regarding usage.
This information is used to shape Storybook's roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://storybook.js.org/telemetry


🔎 found a 'eslintPlugin' migration:
╭ Automigration detected ──────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│                                                                                                                                      │
│   We've detected you are not using our eslint-plugin.                                                                                │
│                                                                                                                                      │
│   In order to have the best experience with Storybook and follow best practices, we advise you to install eslint-plugin-storybook.   │
│                                                                                                                                      │
│   More info: https://github.com/storybookjs/eslint-plugin-storybook#readme                                                           │
│                                                                                                                                      │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
? Do you want to run the 'eslintPlugin' migration on your project? » (Y/n)√ Do you want to run the 'eslintPlugin' migration on your project? ... yes
✅ Adding dependencies: eslint-plugin-storybook
An error occurred while installing dependencies.

❌ error when running eslintPlugin migration
HandledError
    at NPMProxy.addDependencies (C:\Users\malce\AppData\Local\npm-cache\_npx\d28cfb4b54958d89\node_modules\@storybook\cli\dist\generate.js:37:1198)
    at Object.run (C:\Users\malce\AppData\Local\npm-cache\_npx\d28cfb4b54958d89\node_modules\@storybook\cli\dist\generate.js:114:212)
    ... 5 lines matching cause stack trace ...
    at async initiate (C:\Users\malce\AppData\Local\npm-cache\_npx\d28cfb4b54958d89\node_modules\@storybook\cli\dist\generate.js:367:109) {
  handled: true,
  cause: Error
      at NPMProxy.executeCommand (C:\Users\malce\AppData\Local\npm-cache\_npx\d28cfb4b54958d89\node_modules\@storybook\cli\dist\generate.js:39:1699)
      at NPMProxy.runAddDeps (C:\Users\malce\AppData\Local\npm-cache\_npx\d28cfb4b54958d89\node_modules\@storybook\cli\dist\generate.js:39:2894)
      at NPMProxy.addDependencies (C:\Users\malce\AppData\Local\npm-cache\_npx\d28cfb4b54958d89\node_modules\@storybook\cli\dist\generate.js:37:1036)
      at Object.run (C:\Users\malce\AppData\Local\npm-cache\_npx\d28cfb4b54958d89\node_modules\@storybook\cli\dist\generate.js:114:212)
      at runFixes (C:\Users\malce\AppData\Local\npm-cache\_npx\d28cfb4b54958d89\node_modules\@storybook\cli\dist\generate.js:326:1152)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
      at async automigrate (C:\Users\malce\AppData\Local\npm-cache\_npx\d28cfb4b54958d89\node_modules\@storybook\cli\dist\generate.js:321:636)
      at async doInitiate (C:\Users\malce\AppData\Local\npm-cache\_npx\d28cfb4b54958d89\node_modules\@storybook\cli\dist\generate.js:356:373)
      at async withTelemetry (C:\Users\malce\AppData\Local\npm-cache\_npx\d28cfb4b54958d89\node_modules\@storybook\core-server\dist\index.js:35:3422)
      at async initiate (C:\Users\malce\AppData\Local\npm-cache\_npx\d28cfb4b54958d89\node_modules\@storybook\cli\dist\generate.js:367:109)
}

